android/src/main/java/com/rudderstack/react/android/RNRudderSdkModule.java

```diff
@@ -88,6 +89,21 @@ public void screen(String event, ReadableMap properties, ReadableMap options) {
                 .build());
     }
 
+    @ReactMethod
+    public void putAndroidDeviceToken(String token) {
+        if (rudderClient == null) return;
```

Comment:
```
if (rudderClient == null) {
	return;
}
```

For better readability

Reply:
done..ran prettier on all files inside `src`